scikits/odes/sundials/cvode.pxd

```diff
@@ -71,6 +71,22 @@ cdef class CV_WrapJacTimesVecFunction(CV_JacTimesVecFunction):
     cdef int with_userdata
     cpdef set_jac_times_vecfn(self, object jac_times_vecfn)
 
+cdef class CV_ErrHandler:
+    cpdef evaluate(self,
+                   int error_code,
+                   bytes module,
+                   bytes function,
+                   bytes msg,
+                   object user_data = *)
+
+cdef class CV_WrapErrHandler(CV_ErrHandler):
+    cpdef object _err_handler
+    cdef int with_userdata
+    cpdef set_err_handler(self, object err_handler)
+
+cdef class CV_ErrData:
+    cdef CV_ErrHandler err_handler
+    cdef object user_data
 
```

Comment:
Probably some misunderstanding, I thought the conclusion was that it's possible to have
"cdef CV_ErrHandler err_handler" and 
"cdef object user_data" (aka "cdef object err_user_data")
 declared inside of CV_data directly so that we could drop the CV_ErrData class completely. And then we would  pass the CV_data to CVSetErrHandlerFn() instead of the CV_ErrData object. At least that was my intention. So I am wondering, is it possible to include the error handler variables into the CV_data (directly)?

Otherwise all is fine.